.github/workflows/pr-preview.yml

```
@@ -0,0 +1,314 @@
+              2. Look for build errors in the EAS dashboard
+              
+              **Common causes:**
+              - Build configuration issues
+              - Native dependency conflicts
+              - Code signing problems
+              
+              **Build failed at:** ${new Date().toLocaleString()}`
+            })
+
+      - name: Comment on PR - EAS Update
+        if: steps.check-compatibility.outputs.can_create_update == 'true'
```

actionlint (1.7.7)

property "check-compatibility" is not defined in object type {}

(expression)